azurerm/internal/services/containers/container_registry_scope_map_data_source.go

```diff
@@ -0,0 +1,74 @@
+package containers
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceContainerRegistryScopeMap() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceContainerRegistryScopeMapRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"container_registry_name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: ValidateContainerRegistryName,
+			},
+			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"actions": {
```

Comment:
could we add some whitespace here to be consistent with the majority of the provider?
```suggestion
			"container_registry_name": {
				Type:         schema.TypeString,
				Required:     true,
				ValidateFunc: ValidateContainerRegistryName,
			},
			
			"resource_group_name": azure.SchemaResourceGroupNameForDataSource(),
			
			"description": {
				Type:     schema.TypeString,
				Computed: true,
			},
			
			"actions": {
```